stage3.Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN echo "Building Gentoo Container image for ${ARCH} ${SUFFIX} fetching from ${
  && gpg --keyserver ha.pool.sks-keyservers.net --recv-keys ${SIGNING_KEY} \
  && gpg --verify "${STAGE3}.DIGESTS.asc" \
  && awk '/# SHA512 HASH/{getline; print}' ${STAGE3}.DIGESTS.asc | sha512sum -c \
- && tar xjpf "${STAGE3}" --xattrs --numeric-owner \
+ && tar xJpf "${STAGE3}" --xattrs --numeric-owner \
```

Comment:
You should be able to just omit the compression-format (`j`, `J`) entirely - we aren't reading from stdin, and `tar` can just auto-detect this.  That way if it were to change-back `xz` to `bzip2`, this won't need to change again.